### ISSUE TITLE
vizion.analyze fails if no remote named 'origin' is specified

### ISSUE THREAD

**Issue by thewilli** [keymetrics/vizion#12]
if running on a git repository without an origin remote specified, vizion runs into an error:

> Error: Error: Command failed: /bin/sh -c cd '/path/to/repo/';LC_ALL=en_US.UTF-8 git config --get remote.origin.url

Vizion should not crash for repositories without any remote at all or with a remote named other than "origin".

**Comment by Unitech** [comment]
Hello,

Thanks for reporting this issue, does it crashed PM2 too??

**Comment by thewilli** [comment]
@Unitech no, but the pm2-managed app is considered as not being under source control, and no metadata of the git repository is available.

**Comment by Unitech** [comment]
So returning return cb(null, null); here https://github.com/keymetrics/vizion/blob/master/lib/git.js#L20

would do the trick right?
